chore(vscode): search only for `oxfmt` in the first workspace folder

Sysix opened the pull request:
Hoping that with `RelativePattern` the search speed (with symlinks) is better. It required a base path, which will be the first workspace folder

camc314 reviewed (approved):
🤞 thanks for getting to this so quickly

Sysix committed:
chore(vscode): search only for `oxfmt` in the first workspace folder (#16612)

Hoping that with `RelativePattern` the search speed (with symlinks) is better. It required a base path, which will be the first workspace folder
